packages/glaze/src/theme.ts

```diff
@@ -85,19 +85,29 @@ export function fromThemeUI(theme: ThemeUI): StaticTheme {
    */
   function convertBreakpoint(breakpoint: string | number): number {
     if (typeof breakpoint === 'string') {
-      const [, value, unit] = /(\d+)(.+)/i.exec(breakpoint) ?? [];
-      if (unit === 'px') {
-        return Number(value);
-      }
-      if (['rem', 'em'].includes(unit)) {
-        warnOnce(
-          `"${breakpoint}" was converted to ${value}. This could have unintended side-effects.`,
-        );
-        return Number(value) * 16;
+      const [, numberCandidate, unit] =
+        /(.*\d)([a-z]*)/.exec(
```

Comment:
This ignores units like`%` (which is a valid `min-width` unit) and instead simply strips `%` incorrectly translating it into a pixel value

Reply:
Great observation. I think `/(.*\d)([a-z%]*)/` will suffice then, as it cannot be matched to any units later on.